src/auspex/analysis/fits.py

```diff
@@ -1,9 +1,94 @@
 from scipy.optimize import curve_fit
 import numpy as np
 from numpy.fft import fft
+from scipy.linalg import svd, eig, inv, pinv
 from enum import Enum
 
-import matplotlib.pyplot as plt
+def hilbert(signal):
```

Comment:
why not just use scipy.signal.hilbert?

Reply:
The scipy version doesn't take complex inputs, which are used for the cleaned data.

Reply:
true but I think it goes right through and is easier than writing your own:

```
cleaned_analytical_sig = hilbert(np.real(cleaned_data)) +1j * hilbert(np.imag(cleaned_data))
```